### PR TITLE
Install Prop Types Library, Assign Component Prop Types, moved some f…

### PR DESCRIPTION
**Pull request by ofeterovskaya**
Install Prop Types Library, Assign Component Prop Types, moved some folders to the new subfolder components, add and remove buttons are working without refreshing the page.

I didn't fix styles yet, because I was working on fixing add and remove buttons. But I 'll do it till the end of the project

### CODE REVIEW

File: src/components/TodoList.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <ul className={styles.TodoListContent}>
+      {todoList.map((todo) => (
+        <TodoListItem key={todo.id} task={todo} onRemoveTodo={onRemoveTodo} />
+      ))}
+    </ul>
+  );
+};
+
+TodoList.propTypes = {
+  todoList: PropTypes.array.isRequired,
```
Comment (mattcarlotta) [inline src/components/TodoList.js]: Stretch Goal:
What is `todoList` an array of? An array of strings, numbers, or an object (shape) that has structure!? When it comes to arrays and objects, the more specific you are in their structure, the better type checking you'll have during run-time!

<details>
<summary>Click here for answer</summary>

```js
TodoList.propTypes = {
  todoList: PropTypes.arrayOf(
    PropTypes.shape({
      id: PropTypes.string.isRequired,
      title: PropTypes.string.isRequired,
    }),
  ),
  onRemoveTodo: PropTypes.func.isRequired,
}
```
</details>

File: src/components/TodoListItem.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      &nbsp;
+      <button type="button" onClick={() => onRemoveTodo(id)}>
+        Remove
+      </button>
+    </li>
+  );
+}
+
+TodoListItem.propTypes = {
+    onRemoveTodo: PropTypes.func.isRequired,
```
Comment (mattcarlotta) [inline src/components/TodoListItem.js]: Missing Requirement:
The prop `task` needs a type! 

Stretch Goal:
Also, if it’s an object, what is its shape (see answer above if you’re stuck)?

**Review by mattcarlotta — CHANGES_REQUESTED** [review]
Excellent work Oksana, you almost nailed this assignment! You’re just missing one prop type declaration. Once that’s been added, this assignment will be complete.

I added some optional stretch goals if you want to take your code to the next level.

File: src/components/TodoList.js (hunk 1 above)
Comment (ofeterovskaya) [reply src/components/TodoList.js]: Thank you Matt!

**Review by mattcarlotta — APPROVED** [review]

**Pull request merged**